### ISSUE TITLE
Triggers in E12s

### ISSUE THREAD

**Issue by HamelGamel** [quisquous/cactbot#2445]
Cactbot dont say any Triggers for E12s P2 (Oracle). Is this normal? Im DPS and its not working for me. A Friend of mine is Healer and he get Triggers.
I downloaded Cactbot new but it still dont work.
In the settings i can see that there are triggers for Relativities, for the get under and etc., but i dont get any of them.

**Comment by quisquous** [comment]
Are there errors in your OverlayPlugin debug log when this happens?

**Comment by HamelGamel** [comment]
There is no Error in the log. I get the Triggers for P1 but not for Oracle.

**Comment by HamelGamel** [comment]
> Are there errors in your OverlayPlugin debug log when this happens?

There is no Error in the log. I get the Triggers for P1 but not for Oracle

**Comment by Helios747** [comment]
Also having issues with E12S p2 but it's a bit different, I don't know if it warrants it's own issue.

For me some of the triggers just don't fire. Specifically, Basic Relativity just doesn't show any callouts for any debuff but long fire. I have overridden text for the 4. I've also disabled callouts for the yellow hourglass because I suspected they were overwriting the debuff callout, but the behavior is the same.
